src/lib.rs

```diff
@@ -137,6 +137,46 @@ impl<T: EventSubscriber + ?Sized> MutEventSubscriber for T {
     }
 }
 
+impl MutEventSubscriber for Box<dyn MutEventSubscriber> {
+    fn process(&mut self, events: Events, ops: &mut EventOps) {
+        self.deref_mut().process(events, ops);
+    }
+
+    fn init(&mut self, ops: &mut EventOps) {
+        self.deref_mut().init(ops);
+    }
+}
+
+impl MutEventSubscriber for Box<dyn MutEventSubscriber + Send> {
```

Comment:
Does this need to be explicitly implemented?

Reply:
Yes, the boxed series of implementations are added on demand.